### PR TITLE
refs #8432 - fixed readme to match changes to image DO NOT MERGE before ...

### PR DESCRIPTION
**Pull request by orrabin**
https://github.com/theforeman/foreman-discovery-image/pull/1/files in merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,13 +97,14 @@ template and add the following after the "LABEL local" block of options:
...
     KERNEL boot/fdi-image/vmlinuz0
-    APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset foreman.url=http://xyz
+    APPEND initrd=boot/fdi-image/initrd0.img rootflags=loop root=live:/fdi.iso rootfstype=auto ro rd.live.image acpi=force rd.luks=0 rd.md=0 rd.dm=0 rd.lvm=0 rd.bootif=0 rd.neednet=0 nomodeset proxy.url=http://xyz proxy.type=proxy
     IPAPPEND 2
 
-Note the `foreman.url` option on the APPEND line, which defines where the foreman
-instance is. Make sure this is set correctly or discovered hosts will not
-register to Foreman.  Then make this the default by altering the `ONTIMEOUT`
-option:
+Note the `proxy.url` option on the APPEND line,which defines
```
Comment (lzap) [inline README.md]: missing space after the comma

**Comment by lzap** [comment]
I've commented Shlomi's original patch, I think we can simplify it a bit.

**Comment by orrabin** [comment]
@lzap after Shlomi changes the image I'll change this PR according to his example in the image readme file

**Comment by lzap** [comment]
Nitpick - can we tell the user that this is really "Foreman proxy" somewhere. Because proxy is a bit generic term, people can be confused with HTTP(s) proxies. Just insert "foreman" somewhere. Thanks!

**Comment by orrabin** [comment]
@lzap I added foreman-proxy.
Should this wait until https://github.com/theforeman/foreman-discovery-image/pull/2 is merged and add an explanation that the default is foreman? 
right now it says about proxy.type: `Make sure this is set correctly or discovered hosts will not register to Foreman` which was the initial behavior but now it actually defaults to proxy and will change to foreman.

**Comment by lzap** [comment]
Please change it here, yes. I am going to test Gregs patch now. It make sense to be nice for those who forget to change the line. Thanks.

**Comment by orrabin** [comment]
@lzap changed the line about `proxy.type`

**Comment by lzap** [comment]
I've fixed the README, merging. Thank you. cd560e5..c081812

**Pull request closed without merging**